.github/workflows/get-started-php.yml

```diff
@@ -8,11 +8,13 @@ on:
     paths:
       - 'get-started/azure-auth/php/**'
       - 'get-started/quickstart/php/**'
+      - 'msgraph-mail/php/**'
```

Comment:
I'm not sure why we're adding this path?

Reply:
Reverted.